### ISSUE TITLE
unable to replicate results for cifar100

### ISSUE THREAD

**Issue by shairoz** [qubvel/efficientnet#93]
When training efficientnet-b0 on Cifar 100, while the reported results are ~90% accuracy, the model achieves an accuracy of around 7% (Cifar 10 achieves around 40% accuracy).
I've explored several optimizers  (including SGD and RMSprop) with several learning rate schedulers, none seem to affect (used https://github.com/katsura-jp/efficientnet-pytorch/blob/master/cifar100.py as reference).
Any acute modifications has to be made (other than changing the input size and classes)?

additional information:
keras version: 2.2.4
TF version: 1.14.0
Data scaling: [-0.5,0.5]

Thanks

**Comment by nauyan** [comment]
@shairoz The dataset you are using has 100 classes and total training samples are 50000 which approximately makes 500 class per-samples. Due to the above reason this model would not perfectly generalize your dataset therefore it is recommended to use a simpler model for training.

**Comment by ijaz-ia** [comment]
The reason for the reported accuracy to be ~90% of EfficientNet-B0 on CIFAR-100 is that the authors have used transfer learning.

May be by initializing your initial weights with the available pre-trained weights help you.
